sources/packages/web/src/views/institution/AppInstitution.vue

```diff
@@ -20,12 +20,12 @@ export default {
   setup() {
     const router = useRouter();
     const isAuthReady = ref(false);
-
     // Mounding hook
     onMounted(async () => {
       await AppConfigService.shared.initAuthService(ClientIdType.INSTITUTION);
       isAuthReady.value = true;
       const auth = AppConfigService.shared.authService?.authenticated ?? false;
+
```

Comment:
Please consider not committing this file if the only change is the spaces.

Reply:
Done